### PR TITLE
Feature/gallery

### PR DESCRIPTION
**Pull request by annierizvi**

### CODE REVIEW

File: src/Pages/Shop/Shop.css
Comment (Jaycossey) [inline src/Pages/Shop/Shop.css:1]: Came up as empty file, assuming all styles are localised with tailwind

File: src/components/Navbar.jsx
Comment (Jaycossey) [inline src/components/Navbar.jsx:1]: Need to confirm with @arionitas to ensure no funcitonality lost but im happy to approve providing no side effects within the navbar

File: package-lock.json
Comment (Jaycossey) [inline package-lock.json:1]: unable to see any changes here but doesnt seem to be an issue, will let second reviewer to confirm but im happy to proceed

**Review by Jaycossey — APPROVED** [review]
Im happy to approve the changes, if you let the team know in slack and get second approval. Ping me once done and we can get everyone to pull once merged

**Review by arionitas — APPROVED** [review]

**Pull request merged**
